pkg/minishift/addon/manager/addon_manager_test.go

```diff
@@ -197,8 +195,39 @@ func TestRemoveAddon(t *testing.T) {
 	assert.Equal(t, expectedRemoveAddonOutput, tee.StdoutBuffer.String())
 }
 
+func TestRemoveMetaAddon(t *testing.T) {
+	var expectedRemoveMetaAddonOutput = `-- Removing addon 'metaaddon':
+Removing metaaddon without depending on addon default variables
+`
+	path := filepath.Join(basepath, "..", "..", "..", "..", "test", "testdata", "testaddons")
+	manager, err := NewAddOnManager(path, make(map[string]*addon.AddOnConfig))
+	assert.NoError(t, err, "Unexpected error creating manager in directory '%s'", path)
+
+	testaddon := manager.Get("metaaddon")
+	context, _ := command.NewExecutionContext(nil, &FakeSSHDockerCommander{})
+
+	tee := cli.CreateTee(t, false)
+	manager.RemoveAddOn(testaddon, context)
+	tee.Close()
+
+	assert.Equal(t, expectedRemoveMetaAddonOutput, tee.StdoutBuffer.String())
+}
+
 var expectedInvalidAddonOperationError = errors.New("The variable(s) 'TEST' are required by the add-on, but are not defined in the context")
 
```

Comment:
@praveenkumar Wondering why this is declared in between two functions. Can you move it to an appropriate place e.g. beginning of the file?